samples/js/node/common_api_example.js

```diff
@@ -30,6 +30,7 @@ async function run()
   const output = await model.infer(tensor, shape);
 
   //show the results
-  console.log('Result: ' + imagenetClassesMap[math.argMax(output.data)]);
-  console.log(math.argMax(output.data));
+  const result = getMaxElement(output.data);
+  console.log('Result: ' + imagenetClassesMap[result.index],
```

Comment:
Is it possible to use `printClass` function from `./samples/common/helpers.mjs` here?

Reply:
It is possible, but this function generates a warning that importing JSON modules is an experimental feature and might change at any time.
I propose to use this function here after we change it.